### PR TITLE
Closes #1 and plotting script for vtstability

### LINKED ISSUE #1: Update loading of parameters in test/c1c5_vrstability_print.jl
`CubicEoSDatabase.jl` package were updated.

Now `CubicEoSDatabase.Data` module provides access utilities for `CubicEoSDatabase.jl` database sample.

So, you can update `test/c1c5_vtstability_print.jl`

```diff
c1c5mix = load(BrusilovskyEoSMixture,
               names = ("methane", "n-pentane"),
---            comp_physics_db = joinpath(homedir(), ".julia/packages/CubicEoSDatabase/vYhT2/test/example_database/phys/comp/martinez.csv"),
+++            comp_physics_db = Data.martinez(),
---            comp_eos_db = joinpath(homedir(), ".julia/packages/CubicEoSDatabase/vYhT2/test/example_database/eos/comp/brusylovsky.csv"),
+++            comp_eos_db = Data.brusilovsky_comp(),
---            mix_eos_db = joinpath(homedir(), ".julia/packages/CubicEoSDatabase/vYhT2/test/example_database/eos/mix/brusylovsky.csv")
+++            mix_eos_db = Data.brusilovsky_mix()
            )
```

### PR DESCRIPTION
**Pull request by stepanzh**
See 14cfab9.

### CODE REVIEW

File: test/c1c5_vtstability_print.jl [hunk 1]
```diff
@@ -84,4 +86,4 @@ function vt_stability_η_temp(fname::AbstractString, χ_base, Ση_range, T_rang
     end
 end
 
-vt_stability_η_temp("/tmp/c1c5_stability.txt", [0.547413, 0.452587], 100:10:15000, 300:1:450)
```
Comment (stepanzh) [inline test/c1c5_vtstability_print.jl:87]: Newline

File: src/dbload.jl [hunk 2]
```diff
@@ -88,16 +91,23 @@ function __load_brusilovsky_comp__(name::AbstractString,
...
+"""
 function load(::Type{<:BrusilovskyEoSMixture};
               names,
-              comp_physics_db::AbstractString,
-              comp_eos_db::AbstractString,
-              mix_eos_db::AbstractString)
-    # load physical parameters: crit temp, crit press, acentric
+              comp_physics_db::ComponentDatabase=Data.martinez(),
+              comp_eos_db::ComponentDatabase=Data.brusilovsky_comp(),
+              mix_eos_db::MixtureDatabase=Data.brusilovsky_mix())
```
Comment (stepanzh) [inline src/dbload.jl:107]: Default databases are which we used all the time. But, current implementation of `CubicEoSDatabase.Data` do not cache database collection, so each call `Data.martinez()` initialises new object. Same for the others.

I already added an issue on it.

**Comment by stepanzh** [comment]
Update the branch to CubicEoSDatabase v0.2.2, where no collisions between eos and physical source parameters.

**Review by vvpisarev — APPROVED** [review]
Everything works so far, style edits are added as new commits.

**Pull request merged**